package.json

```diff
@@ -31,25 +32,32 @@
   "dependencies": {
     "@changesets/cli": "2.27.10",
     "@manypkg/cli": "^0.21.4",
-    "@sherifforg/create-config": "workspace:*",
```

Comment:
@lishaduck i think this is making the build to fail.

Not sure why you removed this. This is used in the "playground" command, to test the package.